nixos/modules/profiles/installation-device.nix

```diff
@@ -66,6 +66,14 @@ with lib;
 
       Type `sudo systemctl start display-manager' to
       start the graphical user interface.
+    '' +
+    # Upstream bug?
+    # https://github.com/koverstreet/bcachefs-tools/issues/276
+    optionalString (config.boot.supportedFilesystems.bcachefs or false) ''
+
+      If you use encrypted Bcachefs volume, you may need to run
+      `keyctl link @u @s` or use `bcachefs unlock -k session`
+      to have it unlocking successfully.
```

Comment:
I think adding a message to the getty prompt is also not something we should do. There's an infinite number of little pieces of advice we could put here; why should bcachefs be the special one?

I think the `pam_keyinit.so` thing I mentioned before is the real solution here. https://github.com/NixOS/nixpkgs/issues/32279#issuecomment-2726962728

But frankly, the keyring thing is a completely separate issue from this PR and we should tackle it separately.